### ISSUE TITLE
AlertDialog not aligned to center

### ISSUE THREAD

**Issue by serbelga** [JetBrains/compose-jb#1268]
**Description** 

AlertDialog composable is not aligned to center. It requires being inside another composable with a `fillMaxSize()` modifier.

```kotlin
    AlertDialog(
        onDismissRequest = {},
        title = { Text("Title") },
        text = { Text("Text") },
        confirmButton = {
            Button(
                onClick = {}
            ) {
                Text("Confirm")
            }
        },
        dismissButton = {
            Button(
                onClick = {}
            ) {
                Text("Dismiss")
            }
        }
    ) 
```

<img width="640" alt="Captura de pantalla 2021-10-18 a las 18 52 02" src="https://user-images.githubusercontent.com/26246782/137774596-0b70b9e6-6431-4ed3-aaaf-ebc329026c59.png">

```kotlin
    Column(modifier = Modifier.fillMaxSize()) {
        AlertDialog(
            onDismissRequest = {},
            title = { Text("Title") },
            text = { Text("Text") },
            confirmButton = {
                Button(
                    onClick = {}
                ) {
                    Text("Confirm")
                }
            },
            dismissButton = {
                Button(
                    onClick = {}
                ) {
                    Text("Dismiss")
                }
            }
        )
    }
```

<img width="640" alt="Captura de pantalla 2021-10-18 a las 18 53 40" src="https://user-images.githubusercontent.com/26246782/137774799-127f2760-7418-4acb-8fb9-03ce736a38fa.png">

**Library version**
1.0.0-alpha3

**OS**
Mac OS 11 Big Sur

**Comment by akurasov** [comment]
Reproducible on beta5

**Comment by igordmn** [comment]
Should be fixed in 1.0.0-beta6-dev455

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6232">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
